src/client/Microsoft.Identity.Client/AppConfig/AuthorityInfo.cs

```diff
@@ -65,7 +63,7 @@ public AuthorityInfo(
                     CanonicalAuthority = new Uri($"https://{authorityUri.Authority}/{pathSegments[0]}/{pathSegments[1]}/");
 
                     UserRealmUriPrefix = UriBuilderExtensions.GetHttpsUriWithOptionalPort(
-                        $"https://{authorityUri.Authority}/{pathSegments[0]}/common/userrealm/",
+                        $"https://{authorityUri.Authority}/{pathSegments[0]}/{Constants.CommonTenant}/userrealm/",
```

Comment:
Replacing `common` and `organizations` with constants, while it improves maintainability but reduces visibility when debugging.

Reply:
@gladjohn - The general practice over the years that I have seen and followed is to use constants instead of literals/magic values. Does MSAL follows a different guideline?

Reply:
[AuthorityInfo.cs](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/5174#discussion_r1985469332): is a very common code path for all our flows, it is more of a code visibility issue than a coding standard IMHO. Maybe you can rename the constant to `CommonAuthority` from `CommonTenant` or something that makes it more intuitive

Reply:
Done. Renamed to ``Common``, ``Consumers``, and ``Organizations`` similar to in ``TestConstants.cs``.